st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -554,6 +554,13 @@ def _connect(self, host, socket=None):
         if self.key_files:
             conninfo['key_filename'] = self.key_files
 
+            try:
+                paramiko.RSAKey.from_private_key_file(self.key_files)
+            except paramiko.ssh_exception.PasswordRequiredException:
+                msg = ('Private key file %s is passphrase protected. Supply a passphrase!' %
+                       self.key_files)
+                raise Exception(msg)
```

Comment:
The default message in that exception sends out a cryptic message that's not obvious to user. Hence I send out a generic exception.